packages/cli/src/modules/mcp/tools/get-workflow-details.tool.ts

```diff
@@ -126,7 +126,7 @@ export async function getWorkflowDetails(
 	const sanitizedWorkflow: WorkflowDetailsResult['workflow'] = {
 		id: workflow.id,
 		name: workflow.name,
-		active: workflow.active,
+		active: workflow.activeVersionId !== null,
```

Comment:
Not in this PR, but later no we need to decide if we want to include both active and current version for MCP clients